fiftyone/server/routes/frames.py

```
@@ -80,3 +97,59 @@ def run(view):
+        """
+        view = await fosv.get_view(
+            dataset,
+            stages=stages,
+            extended_stages=extended,
+            dynamic_group=dynamic_group,
+            awaitable=True,
+        )
+
+        count = end_frame - start_frame + 1
+
+        def run(view):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `run`

(ANN202)